### ISSUE TITLE
MODxCarbon - Sort Menu Index GIF - Annoying

### ISSUE THREAD

**Issue by modxuser** [modxcms/evolution#599]
media/style/MODxCarbon/images/icons/stick.gif

This image is extremely annoying - it's flashing all the time

Is it not possible to make it static ?

Another point here is accessibility - flashing icons could affect those with seizure problems - https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure.html

**Comment by Deesen** [comment]
I added the sort-menu button but did not search for a proper icon. I did not test MODxCarbon so its really annoying;-). Maybe you can provide an icon we can add to the default themes?

**Comment by modxuser** [comment]
A wide choice: http://www.flaticon.com/search?word=sort

**Comment by modxuser** [comment]
Anyone know what the original icons pack is - the name or download source ?

**Comment by Deesen** [comment]
I found this info: https://modx.com/extras/package/carbonmanagertheme

"Great icons from FamFamFam, CrystalProject and my own"

**Comment by modxuser** [comment]
Ah, thanks - the Silk collection from FamFamFam

Have downloaded the set and will take a look and see what's appropriate

**Comment by modxuser** [comment]
I have taken a look at the icon set and to be honest, there is no perfect fit, BUT:

If you take a look here: http://www.famfamfam.com/lab/icons/silk/previews/index_abc.png

I have picked out a few:

layers
shape_move_back
shape_move_front
table_refresh
table_relationship
wrench
wrench_orange

Maybe one of those is suitable - what do you think ?

**Comment by Deesen** [comment]
I would say "table_refresh" or even "table_sort".  If "table_sort" we could change color of the grey part to make it darker / more clear or so.

**Comment by modxuser** [comment]
![icon](https://cloud.githubusercontent.com/assets/1859868/14458445/7b7b5ff6-00b1-11e6-8d85-94eb820ab862.png)

**Comment by modxuser** [comment]
manager/media/style/MODxCarbon/style.php

Line: 98

`$_style['icons_set_parent']         = $style_path.'icons/table_refresh.png';`

Image:

manager/media/style/MODxCarbon/images/icons/table_refresh.png

![table_refresh](https://cloud.githubusercontent.com/assets/1859868/14458502/fc6e328c-00b1-11e6-87de-91e081ac26a3.png)

**Comment by modxuser** [comment]
I have updated manager/actions/mutate_menuindex_sort.dynamic.php

From line180:
```
if(!$disabled) {
    $header .= '<button onclick="resetSortOrder();" style="float:right;margin-bottom:1em;display:block;">' . $_lang['reset_sort_order'] . '</button>
    <p>' . $_lang["sort_elements_msg"] . ' (<a href="#" onclick="sort();">' . $_lang["sort_alphabetically"] . '</a>)</p>';
};
```
![reset-sort-order-1](https://cloud.githubusercontent.com/assets/1859868/14458850/77457090-00b4-11e6-925f-03934d3c3870.png)

TO:
```
if(!$disabled) {
    $header .= '<ul class="actionButtons"><li><a href="#" onclick="resetSortOrder();">' . $_lang['reset_sort_order'] . '</a></li></ul>
    <p>' . $_lang["sort_elements_msg"] . ' (<a href="#" onclick="sort();">' . $_lang["sort_alphabetically"] . '</a>)</p>';
};
```
![reset-sort-order-2](https://cloud.githubusercontent.com/assets/1859868/14458854/801973ce-00b4-11e6-969f-52e6a44627b4.png)

**Comment by modxuser** [comment]
@fourroses666 

Just updated the code to have the refresh icon as well - see image above
```
if(!$disabled) {
    $header .= '<ul class="actionButtons mis"><li><a href="#" onclick="resetSortOrder();"><img src="'.$_style["icons_refresh"].'" /> ' . $_lang['reset_sort_order'] . '</a></li></ul>
    <p>' . $_lang["sort_elements_msg"] . ' (<a href="#" onclick="sort();">' . $_lang["sort_alphabetically"] . '</a>)</p>';
};
```

**Comment by fourroses666** [comment]
Oké, I have updated the pull.

**Comment by modxuser** [comment]
@fourroses666

Sorry to mess you around - I see in [your pull request](https://github.com/modxcms/evolution/pull/602/commits/af15edcb72f04f1a6ba1c1989838051b3268ad5b) that the class name "mis" is still in the code I modified - it shouldn't be - that was for testing.

This code:
`$header .= '<ul class="actionButtons mis"><li>`

Should be this:
`$header .= '<ul class="actionButtons"><li>`

**Comment by fourroses666** [comment]
No problem, slowly becoming more convenient to Github

**Comment by modxuser** [comment]
Yeh, I havent used it that way before - so thought I would let the DEV team update from my modifications

But sometimes a minor mistake makes for additional work - again, sorry to have caused more work for you

**Comment by fourroses666** [comment]
DEV team still can accept or decline my Pull requests! :+1:
